src/react-ruler/Ruler.tsx

```diff
@@ -0,0 +1,290 @@
+import { convertUnitSize } from '@daybrush/utils';
+import { ref } from 'framework-utils';
+import * as React from 'react';
+
+import { RulerInterface, RulerProps } from './types';
+
+export default class Ruler extends React.PureComponent<RulerProps> implements RulerInterface {
```

Comment:
I'm not sure if you did this, just want to double-check.
We have a prettier and some eslint. Did you try auto-formatting, so it will refactor the code to our current code style